src/Directory.Packages.props

```diff
@@ -7,8 +7,10 @@
     <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.1" />
     <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
     <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
     <PackageVersion Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="7.0.0" />
```

Comment:
Let's add both of these to the dependabot exclusions:

https://github.com/App-vNext/Polly/blob/90c9894ced5bf781bc5229abf377f58206cee42b/.github/dependabot.yml#L18-L27

Reply:
Even though it's only used in tests? I think these can be automatically updated.